css/style.css

```diff
@@ -352,3 +352,69 @@ button:hover {
 .css {
   color: blue;
 }
+
+/* <<<<<Footer>>>>> */
+
+/* footer {
+  background-color: #f5f5f5;
+} */
+
+.contact-form {
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 60px 55px 55px 55px;
+  gap: 20px;
+  background-image: url("../images/form-contact.png");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  border-radius: 100px 0 0 0;
+}
```

Comment:
- [ ] Great job here 👌 I noticed the left corner of the form section doesn't match with the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins), so kindly fix it 👌 

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177458429-6ee207cb-02d7-4b5c-b212-ae4b00b59ef0.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/177458490-609cf4e6-fc89-4321-a15e-fe7ff44e695a.PNG" width="500" /> |

I think this is the only issue you have, Great 🎉
Let me help you 😄
To add two backgrounds you can use the `background` property instead of the `background-image` like this.
```
background: url(../images/form-contact.png), #fff;
```
Then you have to remove the `border-radius`, we don't need it anymore. 👌 that's it 👍